source/drivers/php.txt

```diff
@@ -188,20 +183,6 @@ Language Compatibility
 See Also
 --------
 
-- :ref:`php-libraries-frameworks-and-tools` for working with Drupal,
-  Cake, Symfony, and more from MongoDB
+- :ref:`php-libraries-frameworks-and-tools`
 
 - :ref:`admin-uis`
-
-- MongoDB for the PHP Mind Blog Series:
-
-  - `Part 1: Getting Started <http://blog.mongodb.org/post/24960636131/mongodb-for-the-php-mind-part-1>`_
-
-  - `Part 2: Queries and Indexes <http://blog.mongodb.org/post/26903435041/mongodb-for-the-php-mind-part-2>`_
-
-  - `Part 3: Data Modeling <http://blog.mongodb.org/post/35133050249/mongodb-for-the-php-mind-part-3>`_
-
-.. toctree::
-   :titlesonly:
-
-   /drivers/php-libraries
```

Comment:
Before merging, please confirm if removing this link is kosher given that we already have a `:ref:` to the page above. I'm not sure why this redundant link existed in the first place, but I'd like to make sure the build process was not depending on the `toctree` block here.